node/cli/src/command.rs

```diff
@@ -68,14 +68,13 @@ impl SubstrateCli for Cli {
         2022
     }
 
-    // | --chain           | Description |
-    // |-----------------  |----------- |
-    // | dev               | Four nodes, Four threshold servers, Alice Bob and Charlie and Dave, Development Configuration |
-    // | devnet-local      | Four nodes, four threshold servers, Alice Bob Charlie and Dave, Development Configuration, Docker Compatible |
-    // | integration-tests | Two nodes, Four threshold servers, Alice and Bob, Development Configuration |
-    // | testnet-local     | Two Nodes, Two threshold servers, Alice and Bob, Testnet Configuration, Docker Compatible |
-    // | testnet           | Four nodes, Two threshold servers, Own Seed, Testnet Configuration |
-    // | tdx-testnet       | Four nodes, Four threshold servers, Alice Bob Chalie and Dave, Development Configuration adapted for TDX testnet |
+    // | --chain                            | Description |
+    // |----------------------------------  |----------- |
+    // | dev                                | Four nodes, Four threshold servers, Alice Bob and Charlie and Dave, Development Configuration |
+    // | devnet-local                       | Four nodes, four threshold servers, Alice Bob Charlie and Dave, Development Configuration, Docker Compatible |
+    // | integration-tests                  | Two nodes, Four threshold servers, Alice and Bob, Development Configuration |
+    // | testnet-local                      | Two Nodes, Two threshold servers, Alice and Bob, Testnet Configuration, Docker Compatible |
+    // | <my-testnet>chainspec-inputs.json  | Custom Testnet Configuration with given inputs from a specified JSON file |
```

Comment:
```suggestion
    // | <my-testnet>-chainspec-inputs.json  | Custom Testnet Configuration with given inputs from a specified JSON file |
```

nit: but would be good to establish a little bit of a convention here